LocalPackages/DataBrokerProtection/Sources/DataBrokerProtection/UI/UIMapper.swift

```diff
@@ -451,6 +451,15 @@ fileprivate extension Array where Element == BrokerProfileQueryData {
     }
 }
 
+extension Array where Element == DBPUIScanProgress.ScannedBroker {
+    /// Number of completed broker scans
+    var currentScans: Int {
```

Comment:
I'd be tempted to rename this so the comment isn't necessary. Basically what you wrote in the comment I think would be good (e.g. completeBrokerScansCount)